### ISSUE TITLE
Use of headings is not WCAG 2.0 compliant

### ISSUE THREAD

**Issue by CR96** [Detroit-Urban-Tech-Meetup/zoning-guide-detroit#1]
According to [Web Accessibility Content Guidelines (WCAG) 2.0](https://www.w3.org/TR/WCAG20-TECHS/G141.html), only a single level-one heading should be present on any given page. Additionally, heading levels should only increase by one to properly convey page structure.

Correcting the use of headings will make the website easier to navigate for people using screen-reading software or other assistive web technologies.
